### ISSUE TITLE
New feature: update metadata from file(s)

### ISSUE THREAD

**Issue by billgeo** [linz/lds-metadata-updater#14]
Both Topo and Hydro are keen to expand this to a tool they can use to upload to the LDS from files they have on disk somewhere. 

It will need to be able to update many layers at once so would suggest detecting the correct LDS layer from the metadata '<gmd:URL>' element, e.g. `<gmd:URL>https://data.linz.govt.nz/layer/93652-nz-10m-satellite-imagery-2017/</gmd:URL>` 

It would also be useful to specify the layer URL on the command line, in the case of only updating one metadata file at time, or so they can create a script that loops over layer and metadata files and uploads one at a time.

**Comment by SPlanzer** [comment]
What is the priority of this piece of work? Does it take priority over AIMS and NZGB Plugin testing for 2.18?

**Comment by billgeo** [comment]
No. Not urgent. They have workarounds. Hyrdro are doing an update in June though, so before then would be great. 

To put it in context, it takes them days to do an LDS update at th moment, and a fairly large part of that can be manually updating metadata.
